contracts/strategies/strategy.sol.example

```diff
@@ -0,0 +1,125 @@
+/// @notice SPDX-License-Identifier: MIT
+pragma solidity ^0.6.7;
+
+/// @dev Import the appropriate base contract
+import "../strategy-joe-rush-farm-base.sol";
+
+/// @dev Contracts use NatSpec Format comments. NatSpec documentation is located here: https://docs.soliditylang.org/en/develop/natspec-format.html
+/// @notice The strategy contract for TraderJoe's Token1/Token2 Liquidity Pool with double rewards
+/// @dev If one of the tokens is a bridge extension token (ends with ".e"), the trailing E should be capitalized like so: StrategyJoeAvaxDaiE
+contract StrategyJoeToken1Token2 is StrategyJoeRushFarmBase {
+    /// @dev LP and Token addresses
+    uint256 public token1_token2_poolId = 10;
+    address public joe_token1_token2_lp = 0x1000000000;
+    
+    /// @dev Addresses for tokens not found in the base contracts
```

Comment:
I think in Natspec the dev tag is meant to be used above function headers, not for each inline comment.  That's how I've been using it at least.